tests/stf_writer_test/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.17)
+project(stf_writer_test)
+
+set(DISABLE_STF_DOXYGEN 1)
+set(NO_STF_LTO 1)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+include_directories(
+    ${PROJECT_SOURCE_DIR}/../.. 
+)
+
+add_compile_options(
+    -g
+    -Wall
+    -Wno-parentheses
+    -MMD
+    -D_FILE_OFFSET_BITS=64
+    -D_LARGEFILE_SOURCE
+    -D_GNU_SOURCE
+    -D__STDC_FORMAT_MACROS
+)
+
+add_compile_options($<$<CONFIG:Release>:-Ofast>)
```

Comment:
Wonder if LTO would be good to test for Release?  @bdutro

Reply:
Done.